src/content/topics/integrations/code-references/index.mdx

```diff
@@ -132,6 +132,6 @@ To learn more about `.ldignore` files, read the [`ld-find-code-refs` documentati
 
 The code references feature scans your source code for occurrences of your flag keys. This process requires your source code to reference flag keys exactly as they appear in LaunchDarkly.
 
-However, by default, the React SDK camel-cases all flag keys for easier access with dot notation. To use the code references feature in conjunction with the React SDK, [configure `ld-find-code-refs` to generate camel-case aliases for the flag keys](https://github.com/launchdarkly/ld-find-code-refs/blob/master/docs/ALIASES.md#flag-keys-transposed-to-common-casing-conventions) or configure the React SDK to disable this camel-casing feature.
+However, by default, the React SDK camel-cases all flag keys for easier access with dot notation. To use the code references feature in conjunction with the React SDK, configure `ld-find-code-refs` to generate camel-case aliases for the flag keys, or configure the React SDK to disable this camel-casing feature. To learn more, read [Flag keys transposed to common casing conventions](https://github.com/launchdarkly/ld-find-code-refs/blob/master/docs/ALIASES.md#flag-keys-transposed-to-common-casing-conventions).
```

Comment:
@scribblingfox 
Is this how it should be, or should I pull this "Read more" text into the new line?

Reply:
@levlaz This is great as-is! Thank you!